services/QuillLMS/app/models/segment_integration/activity.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module SegmentIntegration
+  class Activity < SimpleDelegator
+    def common_params
+      {
+        activity_name: name,
+        tool_name: classification.name.split[1]
+      }.reject {|_,v| v.nil? }
+    end
+
+    def content_params
+      {
+        **common_params,
+        concepts: activity_categories.pluck(:name).join(", "),
+        content_partners: content_partners.pluck(:name).join(", "),
+        topic_level_three: topics.select { |topic| topic.level == 3 }[0],
```

Comment:
[Enumerable#find](https://ruby-doc.org/core-3.1.2/Enumerable.html#method-i-find): `topics.find { |topic| topic.level == 3 }` is kind of shorthand for `select { ...}[0]`